app/src/main/java/org/ole/planet/myplanet/service/UploadManager.java

```diff
@@ -96,9 +96,11 @@ public void uploadExamResult(final SuccessListener listener) {
         mRealm.executeTransactionAsync(realm -> {
             List<RealmSubmission> submissions = realm.where(RealmSubmission.class).findAll();
             for (RealmSubmission sub : submissions) {
-                try {
-                    RealmSubmission.continueResultUpload(sub, apiInterface, realm,context);
-                } catch (IOException e) {
+               try {
+                   if(sub.getAnswers().size()>0){
```

Comment:
@leomaxi  if we do this send survey will not work .

Reply:
Send survey from tablet or from server? This is to prevent overwriting peoples survey data and only sync people the person who used the tablet's data. Issue found was we are overwriting everyone's data including those who didn't use the tablet at all.